ppa/pages/models.py

```diff
@@ -120,6 +121,10 @@ class PagePreviewDescriptionMixin(models.Model):
     #: maximum length for description to be displayed
     max_length = 250
 
+    # ('a' is omitted by subsetting and p is added to default ALLOWED_TAGS)
+    #: allowed tags for bleach html stripping in description
+    allowed_tags = bleach.sanitizer.ALLOWED_TAGS[1:] + ['p']
```

Comment:
Is this relying on current order in the allowed tag list? I'd be more comfortable with explicitly filtering it out by tag name. Maybe set logic would be a good solution? (Or maybe it's too verbose)
`list((set(bleach.sanitizer.ALLOWED_TAGS) | set(['p'])) - set(['a']))`